tests/model_hub_tests/models_hub_common/test_performance_model.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+
+import gc
+import os
```

Comment:
Looks not used
```suggestion
```

Reply:
Please apply PyCharm formatting: `Ctrl+A` and `Ctrl+Alt+O` and `Ctrl+Alt+L`

Reply:
Could we use clang format or something else?

Reply:
Yes, any you want. The main thing is just clean-up unused vars and imports, etc.

Reply:
I used autoflake